ansible/group_vars/prod/vars.yml

```diff
@@ -2,6 +2,8 @@
 env: production
 http_port: 80
 
+snmpd_inform_sink: 172.16.238.11
```

Comment:
die snmpd_inform_sink würd ich wieder in die all/vars schieben, so haben wir die mit den selben werten doppelt.